BASE/Test/Microsoft.ApplicationInsights.Test/Microsoft.ApplicationInsights.Tests/Extensibility/TelemetryConfigurationFactorySelfDiagnosticTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.Implementation
+{
+#if !NETCOREAPP1_1
```

Comment:
why do we need this check?

Reply:
haha. because I started this PR before we removed the older versions :)
Thanks for catching that, I'll clean it up.

Reply:
actually, this test class is not needed. it was removed from the solution, but not from the file system. Will delete.